DTS-STN/rules-based-benefit-finder
# Task8 localization

# Description

[Task #8 - Clean up current source code to properly localize](https://taiga.dts-stn.com/project/demoore-rules-based-benefits-finder/us/8?milestone=28). Removed all duplicate results cards with hard coded localization (en,fr), replaced with dynamic i18n. 

## Test Instructions

1. Answer questions on the survey to generate results cards (try multiple different combinations to ensure you see all possible results, including changing province)
1. Click the 'Francais' or 'English' translation button at the top right corner of the page to translate results.
1. Make sure the text strings still match with the official designs. (You can reference the [current production build](https://covid-benefits.alpha.canada.ca/en/start) as a source of truth )

## Checklist
